src/jwt_authorizer/tokens.py

```diff
@@ -91,15 +97,23 @@ def parse_ticket(prefix: str, ticket: str) -> Optional[Ticket]:
     ticket_id, secret_b64 = trimmed_ticket.split(".")
     try:
         int(ticket_id, 16)  # Check hex
-        secret = base64.urlsafe_b64decode(add_padding(secret_b64))
+        secret = base64.b64decode(
+            add_padding(secret_b64), altchars=b"-_", validate=True
+        )
+        if secret == b"":
+            raise ValueError("ticket secret is empty")
         return Ticket(ticket_id=ticket_id, secret=secret)
     except (ValueError, Error) as e:
-        logger.info("Error decoding ticket:", e)
+        logger.info("Error decoding ticket: %s", str(e))
```

Comment:
Does this happen a lot?  While it's good to not spend a lot of cycles with bad auth, I wonder if we might want to bump this up to error level?

Reply:
Or I guess this is one of those functions that is called a lot but failure is normal?

Reply:
Thanks, good catch. Bumping the logging level.